lib/ramble/ramble/workspace/workspace.py

```diff
@@ -281,17 +281,17 @@ def active_workspace():
 
 def get_workspace_path():
     """Returns current directory of ramble-managed workspaces"""
-    path_in_config = ramble.config.get('config:workspace_dirs')
+    path_in_config = ramble.config.get('config:workspace_dirs', scope='user')
```

Comment:
I don't love hard-coding the scope here. I think we should get whatever the correct definition is instead.